superdesk/vocabularies/vocabularies.py

```diff
@@ -168,7 +168,14 @@ class VocabulariesResource(Resource):
         'custom_field_config': {
             'type': 'dict',
             'nullable': True,
-            'schema': {},
+            'schema': {
+                'increment_steps': {
```

Comment:
what this `increment_steps` does?

Reply:
Inside `Settings->metadata->other fields`, add new field and select the Field type = datetime then there you will get the increment steps which actually gives us the flexibility to extend the time of given interval.